libraries/browser-tracker-core/src/tracker/types.ts

```diff
@@ -648,3 +667,18 @@ export interface ClientSession extends Record<string, unknown> {
    */
   firstEventTimestamp: string | null;
 }
+
+export interface ExtendedCrossDomainLinker {
+  domainUserId?: string;
+  /* Timestamp of the cross-domain link click. */
+  timestamp?: number;
```

Comment:
Is `timestamp` required for this type? It looks like the `timestamp` value will always come from [here](https://github.com/snowplow/snowplow-javascript-tracker/pull/1233/commits/e709c9958012e5088e305024beec6ed8a401c3d7#diff-e919d7558f1e654870709d4fbf8ccef9476b13af717811c0c34b5a269052af15R20)

Reply:
This type is only used internally for the creation of the domain values. I moved it around a bit to make it more clear.